Chaincase/App.xaml.cs

```diff
@@ -147,16 +148,17 @@ private static void CurrentDomain_UnhandledException(object sender, UnhandledExc
 
 		public static async Task LoadWalletAsync()
 		{
-			string walletName = Global.Network.ToString();
-			KeyManager keyManager = Global.WalletManager.GetWalletByName(walletName).KeyManager;
+			var global = Locator.Current.GetService<Global>();
```

Comment:
What's the thinking behind this change to DI reference?

Reply:
To eventually break up the `Global` object into more atomic parts, all available through DI

Reply:
oh that makes so much more sense I'm beginning to understand now